website/source/intro/getting-started/acl.html.md

```diff
@@ -43,10 +39,11 @@ path "secret/foo" {
 }
 ```
 
-The policy format uses a longest matching prefix system on the API path
-to determine access control. Since everything in Vault must be accessed
-via the API, this gives strict control over every aspect of Vault, including
-mounting backends, authenticating, as well as secret access.
+The policy format uses a prefix matching system on the API path
```

Comment:
Should we make a note here like "Vault versions prior to 0.1.2 behaved differently, blah blah"